node/src/effect/requests.rs

```diff
@@ -321,33 +321,41 @@ impl Display for StorageRequest {
     }
 }
 
+/// Details of a request for a list of deploys to propose in a new block.
+#[derive(DataSize, Debug)]
+pub struct ListForInclusionRequest {
```

Comment:
Why does it have to be a standalone struct and not an enum variant as it used to be?

Reply:
Ah, I see now. So that it can be a part of `GetChainspecResult`.